nemo/collections/common/metrics/perf_metrics.py

```diff
@@ -33,6 +40,16 @@ class FLOPsMeasurementCallback(Callback):
 
     higher_is_better = True
 
+    def __new__(cls, model_config, *args, **kwargs):
+        create_tensorboard_logger = model_config.get('exp_manager', {}).get("create_tensorboard_logger", False)
+        if not create_tensorboard_logger:
+            logging.warning("TensorBoard Logger should be enabled for computing TFLOPs per sec per gpu")
```

Comment:
logging.warning("FLOPsMeasurementCallback supports only Tensorboard as of now").